better-font-awesome.php

```diff
@@ -487,6 +496,10 @@ public function admin_enqueue_scripts( $hook ) {
 	 * @since  1.0.10
 	 */
 	public function save_options() {
+		if ( false == check_ajax_referer( self::SLUG . '-options', 'bfa_security', false ) ) {
+			wp_die( __( 'Something went wrong. Refresh the page and try again.', 'better-font-awesome' ), 403 );
```

Comment:
Suggestion to clarify the messaging:

```suggestion
			wp_die( __( 'Settings were not saved due to a missing nonce.', 'better-font-awesome' ), 403 );
```

Reply:
Love the `403` btw! TIL.